apps/src/netsim/NetSimShardSelectionPanel.html.ejs

```diff
@@ -0,0 +1,37 @@
+<%
+  var i18n = require('../../locale/current/netsim');
+%>
+<div class="content-wrap">
+  <div class="field-box display-name-control">
+    <label for="netsim_lobby_name"><%= i18n.myName() %></label>
+    <input id="netsim_lobby_name" type="text" value="<%= displayName %>" <% if (displayName.length > 0) { %>disabled<% } %> />
```

Comment:
I was familiar with this convention for classes (and am still slowly correcting throughout netsim), but not for IDs (I've been sticking to snake_case for those).  Do you have a style guide reference for this?